src/en/news/blog/2025/user-accounts-dashboard/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Understanding Ceph User Accounts: Why They Matter & How to Manage Them with Ceph Dashboard"
+date: "2025-06-28"
+author: "Naman Munet"
+categories: "rgw"
+image: "images/ceph-user-accounts.png"
+tags:
+  - "ceph"
+  - "rgw"
+  - "user accounts"
+  - "s3"
+  - "dashboard"
+---
+
+## 1. What’s an Account & Why Use It?
+
+Instead of each S3 user operating independently, an **account groups users and roles under shared ownership**. Resources like buckets and objects are owned by the account, not individual users. This brings major advantages:
+
+- **Aggregated quotas & stats**: Track usage and enforce limits at the account level.
+- **Shared visibility**: Every user or role in the account can list and manage account‑owned buckets.
+- **Streamlined management**: Apply IAM policies centrally within the account for consistent permissions.
+
+---
+
+## 2. Roles & Responsibilities: Understanding Account Users
+
+| Role Type | What It Is | Permissions |
+|-------------------|----------------------------------------------|--------------------------------------------------------------------|
+| **Account Root User** | The master account owner, created by an admin | Full default access to account’s resources; used to define policies and create sub-users |
+| **IAM Users / Roles** | Scoped identities created by the root user | No access by default—they need explicit IAM policies to perform actions |
+
+
+## 3. User Accounts Management
+### 3.1 Account Creation
+
+The account feature is added under dashboard menu Object > Accounts. Once navigated to this page it will show the list of accounts.
+
+![Step 1: Account list page](images/account-list.png)  
+    *Figure: Step 1 - Navigating to Accounts page*
+
+Now we will click on the `Create` button for creating an account.
+
+We will be getting a form to enter values for following fields:
+
+![Step 2.1: Create Account: Enter account details](images/enter-account-form-details1.png)  
+    *Figure: Step 2.1 - Enter account details*
+
+![Step 2.2: Create Account: Enter quota details](images/enter-account-form-details2.png)  
+    *Figure: Step 2.2 - Enter quota details*
+
+After entering the details click on create Account button, post successful account creation you will be redirected to the accounts list page in which we can see the newly created account listed in table, and on expanding the table row, we can see the account details which shows the Account Quota & Bucket Quota for the account.
```

Comment:
After entering the details, click the Create Account button. Upon successful account creation, you will be redirected to the Accounts List page, where the newly created account will appear in the table. Expanding the table row reveals the account details, including the Account Quota and Bucket Quota for the account.